client/packages/common/src/ui/components/portals/AppFooter/AppFooter.tsx

```diff
@@ -29,14 +34,31 @@ export const AppFooter: FC = () => {
 
   const hideFooter = fullScreen || keyboardIsOpen;
 
+  // Colours for the Footer bar, if specified in Store prefs
+  let customColour: string | undefined;
+  let textColour: string | undefined;
+  if (storeCustomColour) {
+    // Try/catch essentially validates the colour string -- if it's invalid, the
```

Comment:
Thanks great having comment like this! Better in the code than just in the code review..